tests/_internal/test_cmake_file.py

```diff
@@ -13,12 +13,12 @@
 class TestRendersCMakeListsFile(TestCase):
     def test_returns_rendered_content(self):
         target_labels = ["foo", "bar"]
+        feature_labels = ["foo", "bar"]
+        component_labels = ["foo", "bar"]
         toolchain_labels = ["baz"]
-        result = render_cmakelists_template(target_labels, toolchain_labels)
+        result = render_cmakelists_template(target_labels, feature_labels, component_labels, toolchain_labels)
 
-        for label in target_labels:
-            self.assertIn(label, result)
-        for label in toolchain_labels:
+        for label in target_labels + feature_labels + component_labels + toolchain_labels:
```

Comment:
Do you think it's worth asserting actual contents of the template here? I.e. compare result with output of an actual render of the template?

`assertIn` is fragile in that it'll pass on potentially unrelated contents. Now, I can't think of a cmake instruction that'd contain `bar` (`CMAKE RHUBARB!`), but it's safer to compare things to the letter where possible.

Reply:
Hmmm, I see your point but I really didn't want to end up having the template replicated in the test. All I really care about for this test is that all the variables that I have passed in appear in the rendered output and that it doesn't error.

Reply:
Ok, since I am mocking I have mocked out at a different point and now can do a proper equality check 👍 
3ed2597